src/app/components/Slideout/Slideout.vue

```diff
@@ -5,31 +5,28 @@
     :class="{ slideoutContainerHidden: !checked, tall:checked}"
   >
     <HamburgerMenuButton v-model="checked" :notification="notifications" :mention="mentions"/>
-    <v-style>
-      .slideout-button:focus {
-        border: {{tabbing ? 'solid white 1px' : 'none !important'}};
-        outline: {{tabbing ? 'solid white 1px' : 'none !important'}};
-      }
-    </v-style>
     <span v-show="checked">
       <span id="slideout-header">
         <SlideoutButton
           :selected="chatSelected"
           name="chat"
           @input="activeTab = 0"
           class='slideout-button'
+          :class="{tabbing: $vxm.chat.tabbing}"
         />
         <SlideoutButton
           name="user"
           :selected="userSelected"
           @input="activeTab = 1"
           class='slideout-button'
+          :class="{tabbing: $vxm.chat.tabbing}"
         />
         <SlideoutButton
           name="settings"
           :selected="settingsSelected"
           @input="activeTab = 2"
           class='slideout-button'
+          :class="{tabbing: $vxm.chat.tabbing}"
```

Comment:
It feels very awkward to me to add `.tabbing` to every single button.
If you decide not to add it to `App.vue`, like I suggested in another comment, perhaps it would make sense to add it to `#slideout-header`?